.github/actions/post_to_teams/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Post to teams
+description: Post a message to the teams channel
+
+inputs:
+  webhook:
+    description: "The webhook to send the message to."
+    required: true
+  message:
+    description: "The message to be sent."
+    required: true
+
+outputs:
+  commit-msg:
+    description: "Commit message"
+    value: ${{ steps.bump-version.outputs.commit-msg }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Post to teams
+      shell: bash
+      run: |
+        curl \
+          -H "Content-Type: application/json" \
+          -d '{"workflow_name": "${{github.workflow}}", "job_id": "${{github.job}}", "run_id": "${{github.run_id}}", "ref": "${{github.ref_name}}", "message": "${{inputs.message}}"}' \
```

Comment:
Can we quote the message string, or at least document what it may and may not contain?

Reply:
Seems like quoted strings just disappear from the message, should I just document this in the description of the workflow run?

Reply:
Documented this behavior in the description of the `message` parameter.